packages/react-card/src/components/Card/useCard.ts

```diff
@@ -13,18 +31,21 @@ import { useFocusableGroup } from '@fluentui/react-tabster';
  * @param ref - reference to root HTMLElement of Card
  */
 export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>): CardState => {
+  const { appearance = 'filled', focusable = false } = props;
+
   const groupperAttrs = useFocusableGroup({
-    tabBehavior: 'limitedTrapFocus',
+    tabBehavior: getFocusStrategy(focusable),
   });
 
-  const { appearance = 'filled' } = props;
   return {
     appearance,
+    focusable,
 
     components: { root: 'div' },
     root: getNativeElementProps(props.as || 'div', {
       ref,
       role: 'group',
+      tabIndex: focusable !== false ? 0 : undefined,
```

Comment:
Another argument for getting rid of the boolean union, this would be much easier:

```suggestion
      tabIndex: focusable ? 0 : undefined,
```